src/libs/utils.js

```diff
@@ -13,6 +13,13 @@ export const applyHoverEffects = (e, style) => {
   });
 };
 
+export const highlightExactMatches = (highlightedWords, content) => {
+  const regexWords = highlightedWords.map(w => '\\b' + w + '\\b');
+  const regexString = '(' + regexWords.join('|') + ')';
+  const regex = new RegExp(regexString, 'gi');
+  return content.replace(regex, "<span style=\"background-color: yellow\">$1</span>");
```

Comment:
Codacy warning here is a non-issue since we are sanitizing the content that gets rendered.